markdown/3.x/en/ff-record-list.md

```diff
@@ -141,5 +141,7 @@ We recommend using higher values for `[infinite-debounce-delay]` and `infinite-s
     </ff-record>
 </ff-record-list>
 ```
+**Note:**
+Please do not use `infinite-scroll` in `ff-record-list` inside components like `ff-recommendation` or `ff-pushed-products`. This attribute is not designed to work with them and using it may cause unwanted side effect
```

Comment:
Please consider applying the textual changes from the `1.x` Notes.

Reply:
changed